### PR TITLE
Use correct GIC SPIs for EMMC and SDHost controllers.

### PR DESCRIPTION
**Pull request by jaredmcneill**

### CODE REVIEW

**Comment by pbatard** [comment]
Thanks. Be mindful however that, now that the `pi4_dev4` branches we have here have pretty much been fully integrated in mainstream edk2-platforms (especially, `Sdhc.asl` is the same as the one you find here), there is no guarantee that we will act as brokers and upstream the patch for you. Apart from the binary builds, everything you see here should be considered transient and should not be used as a base for development unless you don't care about your work being integrated. Instead, you should strive to use mainline edk2-platforms as your base.

In other words, if you want a patch applied to the RPi4 platform, you should follow the regular edk2/edk2-platforms submission process and submit a patch for mainline on the edk2-devel mailing list.

**Comment by jaredmcneill** [comment]
Will do, thanks!

**Comment by pbatard** [comment]
Thanks for understanding.

Note that the edk2 submission process can be a bit demanding, as opposed to GitHub pull requests. You'll probably want to look at [this guide](https://github.com/tianocore/tianocore.github.io/wiki/Laszlo%27s-unkempt-git-guide-for-edk2-contributors-and-maintainers) that explains how to submit a patch to edk2, knowing that what applies to edk2 also applies to edk2-platforms.

**Comment by andreiw** [comment]
FWIW these are long overdue fixes to the whole set of the interrupt vectors here for "legacy" devices and I'm really grateful to the NetBSD team and Jared in particular for providing fixes for these.
